common/src/state/call.rs

```diff
@@ -197,6 +212,12 @@ impl Call {
         }
     }
 
+    fn remove_participants(&mut self, id: DID) {
```

Comment:
the function name should use the singular noun  - `remove_participant` unless it takes a `Vec` or similar.